src/test/scala/ClientSpec.scala

```diff
@@ -231,6 +236,38 @@ class ClientSpec extends Specification with NoTimeConversions {
       url must contain("%5B%22abc%22,%22def%22%5D")
     }
 
+    "queue a single event" in {
+
+      client.queueEvent(
+        collection = "foo",
+        event = """{"foo": "bar"}"""
+      )
+      true must beEqualTo(true)
```

Comment:
I think these are stub to-do tests, unless you're meaning to assert that there is no exception? There are some explicit ways that that could be expressed.